UnityProject/Assets/Scripts/Managers/AudioManager.cs

```diff
@@ -25,6 +25,23 @@ public class AudioManager : SingletonManager<AudioManager>
         public AudioMixerGroup AmbientMixer;
         public AudioMixerGroup TTSMixer;
 
+        private float GameplayVolumeLevel = 1;
+
+        public float gameplayVolumeLevel
+        {
+	        set
+	        {
+		        if (value > 1) //No earap please
```

Comment:
lmao what happened that made you do this?

Reply:
There has been two reported instances of the jukebox having its audio above 100%

Reply:
well, going to work on ears, e.g being in space Makes it so you can't hear stuff